test/e2e/e2e_test.go

```diff
@@ -240,24 +210,145 @@ func TestLowNodeUtilization(t *testing.T) {
+	if err != nil {
+		t.Fatalf("Unable to create load balancing pods: %v", err)
+	}
+	defer cleanUp()
+
+	t.Log("Creating pods each consuming 10% of node's allocatable")
+	nodeCpu := workerNodes[0].Status.Allocatable[v1.ResourceCPU]
+	tenthOfCpu := int64(float64((&nodeCpu).MilliValue()) * 0.1)
+
+	t.Log("Creating pods all bound to a single node")
+	for i := 0; i < 5; i++ {
+		pod := &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      fmt.Sprintf("lnu-pod-%v", i),
+				Namespace: testNamespace.Name,
+				Labels:    map[string]string{"test": "node-utilization", "name": "test-rc-node-utilization"},
+			},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{{
+					Name:            "pause",
+					ImagePullPolicy: "Never",
+					Image:           "kubernetes/pause",
+					Ports:           []v1.ContainerPort{{ContainerPort: 80}},
+					Resources: v1.ResourceRequirements{
+						Limits: v1.ResourceList{
+							v1.ResourceCPU: *resource.NewMilliQuantity(tenthOfCpu, resource.DecimalSI),
+						},
+						Requests: v1.ResourceList{
+							v1.ResourceCPU: *resource.NewMilliQuantity(tenthOfCpu, resource.DecimalSI),
+						},
+					},
+				}},
+				Affinity: &v1.Affinity{
+					NodeAffinity: &v1.NodeAffinity{
+						RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
+							NodeSelectorTerms: []v1.NodeSelectorTerm{
+								{
+									MatchFields: []v1.NodeSelectorRequirement{
+										{Key: "metadata.name", Operator: v1.NodeSelectorOpIn, Values: []string{workerNodes[0].Name}},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+
+		t.Logf("Creating pod %v in %v namespace for node %v", pod.Name, pod.Namespace, workerNodes[0].Name)
+		_, err := clientSet.CoreV1().Pods(pod.Namespace).Create(ctx, pod, metav1.CreateOptions{})
+		if err != nil {
+			t.Logf("Error creating LNU pods: %v", err)
+			if err = clientSet.CoreV1().Pods(pod.Namespace).DeleteCollection(ctx, metav1.DeleteOptions{}, metav1.ListOptions{
+				LabelSelector: labels.SelectorFromSet(labels.Set(map[string]string{"test": "node-utilization", "name": "test-rc-node-utilization"})).String(),
+			}); err != nil {
+				t.Fatalf("Unable to delete LNU pods: %v", err)
+			}
+			return
+		}
+	}
+
+	t.Log("Creating RC with 5 replicas owning the created pods")
+	rc := RcByNameContainer("test-rc-node-utilization", testNamespace.Name, int32(5), map[string]string{"test": "node-utilization"}, nil, "")
 	if _, err := clientSet.CoreV1().ReplicationControllers(rc.Namespace).Create(ctx, rc, metav1.CreateOptions{}); err != nil {
-		t.Errorf("Error creating deployment %v", err)
+		t.Fatalf("Error creating RC %v", err)
+	}
+	defer deleteRC(ctx, t, clientSet, rc)
+	waitForRCPodsRunning(ctx, t, clientSet, rc)
+
+	// Run LowNodeUtilization strategy
+	evictionPolicyGroupVersion, err := eutils.SupportEviction(clientSet)
+	if err != nil || len(evictionPolicyGroupVersion) == 0 {
+		klog.Fatalf("%v", err)
```

Comment:
Should this be `t.Fatalf()` instead of using `klog`?

Looks like this is one of three places where `klog` is used in the e2e tests. Not sure why it is necessary use `klog` in the e2e tests.

Reply:
> Should this be t.Fatalf() instead of using klog?

Yes, thanks for noticing. Done.